superset/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -186,7 +186,6 @@ class ChartContainer extends React.Component {
           id={this.props.containerId}
           ref={(ref) => { this.chartContainerRef = ref; }}
           className={this.props.viz_type}
-          style={{ overflowX: 'auto' }}
```

Comment:
I did a bit of code archeology for this as I remembered fixing the call. `overflow-x` was introduced here https://github.com/airbnb/superset/commit/2d0ebeae1bfd5e385000c9aa952891cf474821c6, but wasn't valid React style, so I fixed the syntax here https://github.com/airbnb/superset/commit/c1558578d7c555fb7bb9ee30eb98bad4d28fbd07 , but by fixing it I really just brought something we didn't want.

Here are example of viz that override to `overflow-x` to set it to `auto`:
https://github.com/airbnb/superset/blob/master/superset/assets/visualizations/nvd3_vis.css#L15
https://github.com/airbnb/superset/blob/master/superset/assets/visualizations/nvd3_vis.css#L28